doc/mgr/influx.rst

```diff
@@ -0,0 +1,188 @@
+=============
+Influx Plugin 
+=============
+
+The influx plugin continuously collects and sends time series data to an influxdb database. Users have the option to specify what type of stats they want to collect. 
+Some default counters are already set. However, users will have the option to choose some additional counters to collect 
+
+-------------
+Configuration 
+-------------
+
+In order for this module to work, the following configuration should be created ``/etc/ceph/influx.conf``
+
+^^^^^^^^
+Required 
+^^^^^^^^
+
+The configurations must include the following under the header ``[influx]``
+
+:Configuration: **Description**
+:interval: Sets how often the module will collect the stats and send it to influx
+:hostname: Influx host
+:username: Influx username
+:password: Influx password
+:database: Influx data base, if a database doesn't alread exist in influx the module will create one 
```

Comment:
@mhdo2 It would be good if we can use `does not` instead of `doesn't`
Kindly have a look at PR: https://github.com/ceph/ceph/pull/16705